strategies/panoptic_getcollateral_example.py

```diff
@@ -53,6 +53,7 @@ async def async_task(self):
         request_payload["riskPartner"] = 0
         request_payload["strike"] = 100
         request_payload["width"] = 10
+        request_payload["panopticPool"] = "0xc34C41289e6c433723542BB1Eba79c6919504EDD" #T0/T1 PanopticPool 
```

Comment:
Nice - should this be in the other examples too or is this the only example we're maintaining at the moment?

Reply:
Just maintaining this example for now, but I have a card in to break into multiple example scripts and will add it to those as appropriate.